packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.module.scss

```diff
@@ -0,0 +1,7 @@
+:local {
+  .long-text-example {
+    p:first-child {
+      margin-top: 0;
+    }
```

Comment:
can also add no margin-bottom on last `<p>` so it is brings the show/hide button closer to the text and saves some extra vertical height.
```
    p:last-child {
      margin-bottom: 0;
    }

```